scripts/suspend.py

```diff
@@ -126,27 +126,25 @@ def suspend_nodes(nodes: List[str]) -> None:
 def main(nodelist):
     """main called when run as script"""
     log.debug(f"SuspendProgram {nodelist}")
-    nodes = util.to_hostnames(nodelist)
 
     # Filter out nodes not in config.yaml
-    cloud_nodes, local_nodes = lkp.filter_nodes(nodes)
-    if len(local_nodes) > 0:
-        log.debug(
-            f"Ignoring slurm-gcp external nodes '{to_hostlist_fast(local_nodes)}' from '{nodelist}'"
-        )
-    if len(cloud_nodes) > 0:
+    other_nodes, pm_nodes = separate(
```

Comment:
nit: s/other_nodes/npn_nodes ? and may be adding comment above 
npn=non-power-managed
pn=power-managed